src/Cake.AliaSql/Cake.AliaSql.csproj

```diff
@@ -3,9 +3,31 @@
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <AllowedOutputExtensionsInPackageBuildOutputFolder>$(AllowedOutputExtensionsInPackageBuildOutputFolder);.pdb</AllowedOutputExtensionsInPackageBuildOutputFolder>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <Version>0.0.0</Version>
+    <Authors>Richard Simpson</Authors>

```

Comment:
Can we create a follow up issue to set this to use NuGet.org usernames, as per docs here: https://docs.microsoft.com/en-us/nuget/reference/nuspec#authors

Reply:
Done: https://github.com/cake-contrib/Cake.AliaSql/issues/38